alpha/apps/kaltura/lib/batch2/bcdl/kBusinessPreConvertDL.php

```diff
@@ -423,6 +423,13 @@ public static function decideAddEntryFlavor(BatchJob $parentJob = null, $entryId
 			KalturaLog::err("Flavor Params Id [$flavorParamsId] not found");
 			return null;
 		}
+		
+		//Check if the flavor we are trying to generate has source flavor, source can only be ingested, not generated.
+		if(in_array(assetParams::TAG_SOURCE, $flavorParams->getTagsArray()))
+		{
+			KalturaLog::err("Cannot generate flavor params Id [$flavorParamsId], flavor has source tag");
```

Comment:
Using kalturaLog::err is wrong here, since it generate exception traceback message , and it is not required.
I would change it to -  warning or notice.